style/scss/helpers/_mixins.scss

```
@@ -1,20 +1,15 @@
-@mixin small-icon-size {
-    width: 20%;
-    height: 20%;
+@mixin square ($size) {
```

Nadmiarowa spacja : )
```suggestion
@mixin square($size) {
```